### DISCUSSION TITLE
prepare geo_em file for Noah-MP MMF groundwater and crop scheme simulation

### DISCUSSION THREAD

**Discussion by cenlinhe** [NCAR/hrldas#232] [Q&A]
Response from an email chain answering users' question about creating geo_em file for Noah-MP MMF groundwater scheme simulation:

you need to link GEOGRID.TBL.ARW.noamp to use additional parameters (related to groundwater and crop) in WPS geogrid. There isn't a documentation for this procedure, but some information are scattered around the WRF forum and WPS update page: 
https://forum.mmm.ucar.edu/threads/difference-between-geogrid-tbl-arw-and-geogrid-tbl-noahmp.259/
https://www2.mmm.ucar.edu/wrf/users/wps_files/wpsv3.9/updates-3.9.html

To use this table in WRF v3.9 and later, you would need to link GEOGRID.TBL.ARW.noamp with the default table in the WPS/geogrid/ directory:

`ln -sf  GEOGRID.TBL.ARW.noamp  GEOGRID.TBL`

Then run ./geogrid.exe will give you these additional parameters in geo_em.d01.nc. 
You can read these parameter information in  GEOGRID.TBL.ARW.noamp.

**Comment by tslin2** [comment]
additional instruction: https://github.com/NCAR/hrldas/discussions/97
